src/MongoDB/Manager.c

```diff
@@ -58,8 +58,12 @@ PHP_METHOD(Manager, __construct)
 {
 	php_phongo_manager_t     *intern;
 	zend_error_handling       error_handling;
+#if PHP_VERSION_ID >= 70000
+        zend_string              *uri_string;
+#else
```

Comment:
I had a crash at this line when I used 's' and according to the core file the uri_string (char*) version was garbage. This on the other hand worked.